ui/src/app/components/cases/test-case-details.component.ts

```diff
@@ -239,6 +239,43 @@ export class TestCaseDetailsComponent extends BaseComponent implements OnInit {
       }
     }
   }
+  public fetchLinkedTestCases(stepGroupId) {
+    let testCases: InfiniteScrollableDataSource;
+    testCases = new InfiniteScrollableDataSource(this.testCaseService, "workspaceVersionId:" + this.version.id + ",deleted:false,stepGroupId:" + stepGroupId);
+    waitTillRequestResponds();
+    let _this = this;
+
+    function waitTillRequestResponds() {
+      if (testCases.isFetching)
+        setTimeout(() => waitTillRequestResponds(), 100);
+      else {
+        _this.openStepGroupDeleteDialog(testCases);
+      }
+    }
+  }
+
+    private openStepGroupDeleteDialog(list) {
+    this.translate.get("message.common.confirmation.default").subscribe((res) => {
+      const dialogRef = this.matModal.open(ConfirmationModalComponent, {
+        width: '450px',
```

Comment:
plz have this width configurable to avoid duplication in design

Reply:
i dont understand what configurable width mean. width of 450px was also implemented in cloud

Reply:
configuration means, the modal popup styling + visual things can be made as exportable configuration which can be reused in multiple places which shares similar visual